<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceVeiwerWidget</class>
 <widget class="QWidget" name="InvoiceVeiwerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="title">
      <string>السند</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWebView" name="webView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>حفظ السند </string>
         </property>
         <property name="icon">
          <iconset resource="BatteryManagement.qrc">
           <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printButton">
         <property name="text">
          <string>طباعة السند</string>
         </property>
         <property name="icon">
          <iconset resource="BatteryManagement.qrc">
           <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>إلغاء الأمر</string>
         </property>
         <property name="icon">
          <iconset resource="BatteryManagement.qrc">
           <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BatteryManagement.qrc"/>
 </resources>
 <connections/>
</ui>
